### PR TITLE
add popup

### PR DESCRIPTION
**Pull request by zadorogniyandrey**
### Popup

[Demo](https://zadorogniyandrey.github.io/kottans-popup/) |
[Code base](https://github.com/zadorogniyandrey/kottans-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Review by kasionio — CHANGES_REQUESTED** [review]
Good job. 
Please remove redundant files like icons, left just files needed for review.
Too big margins for my small screen :)
Why text below the icons display one time just under App icon?

Screen Shot 2022-08-05 at 20 37 42

**Comment by zadorogniyandrey** [comment]
Thanks! made a change

**Comment by zadorogniyandrey** [comment]
Hi! Now right?

File: submissions/zadorogniyandrey/index.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (kasionio) [inline submissions/zadorogniyandrey/index.html:7]: Please add corresponding title

File: submissions/zadorogniyandrey/index.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    <link rel="stylesheet" href="css/style.min.css">
+</head>
+<body> 
+    <div class="wrapper">
+        <header class="header">
+            <div class="header__links">
```
Comment (kasionio) [inline submissions/zadorogniyandrey/index.html:13]: this is semantically and logically is also a list

File: submissions/zadorogniyandrey/index.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            </div>
+            <div class="header__info">
+                <input class="menu__input" id="popup" type="checkbox">
+                <label class="header__info-item menu" for="popup">
+                    <img class="menu-img" src="./img/menu.png" alt="menu">  
+                </label>
+                <div class="popup">
+                    <ul class="popup__items">
+                        <li class="popup__item">
+                            <a href="" class="popup__link">
```
Comment (kasionio) [inline submissions/zadorogniyandrey/index.html:25]: a href="#"

File: submissions/zadorogniyandrey/scss/style.scss [hunk 4]
```diff
@@ -0,0 +1,178 @@
+@import "nullstyle.scss";
+
```
Comment (kasionio) [inline submissions/zadorogniyandrey/scss/style.scss:2]: Try to use relative units `rem` and `em`, it's a good practice nowadays. 
You can read articles [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units), [here](https://www.freecodecamp.org/news/what-is-rem-in-css/) or any related to it's info

**Review by kasionio — CHANGES_REQUESTED** [review]
The container is too big, so you have a big top margin which doesn't look nice
image

Please make the items container to be equal in width and height. Remove link `a` styling in text, you can give black color to the font
image

Add cursor pointer to `More` button on hover and popup icon. Make padding from item to container equal
image

**Comment by zadorogniyandrey** [comment]
Thanks for the extended comment. Bugs already fixed

**Comment by zadorogniyandrey** [comment]
> The container is too big, so you have a big top margin which doesn't look nice image
> 
> Please make the items container to be equal in width and height. Remove link `a` styling in text, you can give black color to the font image
> 
> Add cursor pointer to `More` button on hover and popup icon. Make padding from item to container equal image

I can’t make the same indent, because it depends on the height of the image

**Comment by zadorogniyandrey** [comment]
img

I can’t make the same indent, because it depends on the height of the image

**Comment by zadorogniyandrey** [comment]
I understood, I made the same indent from the container

File: submissions/zadorogniyandrey/scss/style.scss (hunk 4 above)
Comment (kasionio) [reply submissions/zadorogniyandrey/scss/style.scss:2]: I still can see only pixels :)

File: submissions/zadorogniyandrey/scss/style.scss [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+    border: 2px solid rgb(94, 170, 233);
+}
+.popup__img-container{
+    display: flex;
+    align-items: center;
+    flex-grow: 1;
+}
+.popup__img{
+    width: 100%;
+    top: -50%;
```
Comment (kasionio) [inline submissions/zadorogniyandrey/scss/style.scss:140]: please explain why do you need this

File: submissions/zadorogniyandrey/scss/style.scss [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+.popup__items{
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 10px;
+    align-items: center;
+    margin-bottom: 10px;
+}
+.popup__item{
+
```
Comment (kasionio) [inline submissions/zadorogniyandrey/scss/style.scss:113]: empty line

File: submissions/zadorogniyandrey/scss/style.scss [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+    border-radius: 5px;
+    padding: 10px;
+    box-shadow: 0px 0px 9px rgb(146, 146, 146);
+    display: none;
+    overflow-y: auto;
+}
+.popup__items{
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (kasionio) [inline submissions/zadorogniyandrey/scss/style.scss:107]: grid-template-columns: repeat(3, 1fr);

**Review by kasionio — CHANGES_REQUESTED** [review]
Please leave in PR just files that need to be reviewed, remove icons and images. And fix the top margin
image

File: submissions/zadorogniyandrey/scss/style.scss (hunk 5 above)
Comment (zadorogniyandrey) [reply submissions/zadorogniyandrey/scss/style.scss:140]: > please explain why do you need this

Comment (zadorogniyandrey) [reply submissions/zadorogniyandrey/scss/style.scss:140]: > please explain why do you need this

initially the picture was not in the container and I tried to make it vertically in the center

**Comment by zadorogniyandrey** [comment]
Translated to em margin and padding. Width and height, as I understand it, should be left in px?

**Review by kasionio — CHANGES_REQUESTED** [review]
Nice. But still redundant files in PR

**Comment by zadorogniyandrey** [comment]
did i have to remove scss?)

**Comment by kasionio** [comment]
> did i have to remove scss?)

No, scss is needed. Go to Files Changed tab, there you can check all the files

**Comment by zadorogniyandrey** [comment]
I don't understand what is redundant( All pictures are used

**Comment by kasionio** [comment]
> I don't understand what is redundant( All pictures are used



> I don't understand what is redundant( All pictures are used

Please read carefully the description of the task, you shouldn't have any files besides that need to be reviewed. How pictures are used here?

**Comment by zadorogniyandrey** [comment]
deleted all pictures

**Comment by zadorogniyandrey** [comment]
if I understood correctly

**Comment by kasionio** [comment]
> if I understood correctly

Let's check :D I just want to be sure that you understand how it works. So, why you thought that you need pictures here?

**Comment by zadorogniyandrey** [comment]
I thought you should see what my work looks

**Comment by kasionio** [comment]
> I thought you should see what my work looks

Yes, I should see the result in demo and I still can see it, right?

**Comment by zadorogniyandrey** [comment]
Thank you I understand:)

**Comment by kasionio** [comment]
And now let's check also css folder. Again. Do we need it here?

**Comment by zadorogniyandrey** [comment]
No, still min.css is not readable

**Review by kasionio — APPROVED** [review]
Congrats. Let's move to real programming, JS is waiting :)

**Comment by zadorogniyandrey** [comment]
Really?:) thanks for the help!!!

**Comment by kasionio** [comment]
> Really?:) thanks for the help!!!

You are welcome :)

**Pull request merged**